one_day_intern/settings.py

```diff
@@ -44,6 +44,7 @@
 
     'main',
     'users',
+    'assessee',
```

Comment:
Arya, is it possible to move the assessee functionality under the 'user' app that I have implement, such that all the login and registration along with other authorization functionalities will be placed under 1 app, and that the urls will have the same prefix.